devel/howdoi/Portfile

```diff
@@ -4,33 +4,27 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                howdoi
-version             1.1.9
+version             2.0.19
 categories          devel python
-platforms           darwin
 license             MIT
 supported_archs     noarch
-
-python.default_version 27
-
 maintainers         nomaintainer
 
 description         instant coding answers via the command line
 long_description    {*}${description}
 homepage            https://pypi.python.org/pypi/howdoi
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
-distname            ${python.rootname}-${version}
-
-checksums           rmd160  9c1fdc3dfd03fd717b4978b05928bc0d565f6f10 \
-                    sha256  2b4a0ebc39c9e0489a17ef73e3eec7dd4a2975e07b995406887c639481da2bb5
+checksums           rmd160  bb99d6f9638be1ea3e3e51bca244e1c66ef76164 \
+                    sha256  4baa5270a98e9d001094d873a5a1b6a32eac021009cfc40cd9aeede633cb7556 \
+                    size    27109
 
-depends_build-append \
-    port:py${python.version}-setuptools
+python.default_version 39
```

Comment:
please test with Python 3.10 and if it works stick with that version

Reply:
> please test with Python 3.10 and if it works stick with that version

https://github.com/macports/macports-ports/pull/14826#issuecomment-1125311217

Reply:
why are we back to Python 3.9 again?

Reply:
@reneeotten Several dependencies are not available for `python310`.